Configurations/10-main.conf

```diff
@@ -1718,6 +1718,7 @@ my %targets = (
                                    debug   => "-g -O0",
                                    release => "-O3"),
         lib_cppflags     => "-DTERMIOS -DL_ENDIAN",
+        ex_libs          => add("-lcrypt32"),
```

Comment:
Why this?

Reply:
Coole be deleted. Was a workaround to get the build work.